README.md

```diff
@@ -115,6 +124,27 @@ file, they can be found in [`scripts`](./docker/package/scripts) and
 [`defaults`](./docker/package/defaults) folders respectively.
 
 
+### Multiple similar systemd services
+
+It's possible to run multiple same services, e.g. two `tezos-node`s that run different
+networks.
+
+`tezos-node` packages provide three services out of the box:
+`tezos-node-delphinet`, `tezos-node-carthagenet` and `tezos-node-mainnet` that run
+`delphinet`, `carthagenet` and `mainnet` networks respectively.
```

Comment:
Same question as above: do we support custom networks? It's fine to require some manual configuration for custom networks, but it should be described.

Reply:
No, we don't support them out of the box. Also, I'd say that currently, it's hard to support them via the existing node service structure, because the `tezos-node-start` startup script does `config init/update` and AFAIU, it's not possible to easily make node use some custom network via this command and the node config file should be edited manually

Reply:
AFAIU in order to run node in a network other than these three one has to:
1. Specify a different data dir.
2. Edit the configuration manually.

I see that `docker/package/defaults/tezos-node.conf` has been removed, so now it's harder to do that, but can we at least describe a way to do it manually? Maybe in a separate PR.

Reply:
the problem is that `tezos-node-start` will try to override the config, so this startup script will require additional changes as well

Reply:
Meh, ok, but it was impossible before this PR as well, right? If yes, I propose to create a new issue about supporting custom networks. Can you please do it?

Reply:
See #99